generate-certs/main.tf

```diff
@@ -37,8 +36,8 @@ resource "aws_iam_role_policy" "s3_gen_certs" {
         "s3:PutObject"
       ],
       "Resource": [
-        "arn:aws:s3:::${replace(var.s3_root_path,"/(\/)+$/","")}",
```

Comment:
Strangest thing, I cannot get Terraform to allow this now. Getting the following:

```
Error loading Terraform: Error downloading modules: module openvpn_server: Error parsing .terraform/modules/bf21773e112cafd6226a9e0ab47e5ce3/main.tf: At 193:59: illegal char escape
```